tag_manager/authorized_api/src/triggers.yaml

```diff
@@ -14,6 +14,16 @@ paths:
 
         get:
             summary: List of triggers
+            description: |
+                ### JSON API compatibility divergence
+
+                #### Filters
+                In JSON API standard (https://jsonapi.org/recommendations/#filtering) one can request elements with multiple
+                filter values by simply providing them separated by coma:
+
+                `triggers?filter[trigger_type]=click,event`
+
+                Unfortunately it is not possible in this endpoint.
```

Comment:
I just want to make sure that this endpoint won't accept multiple values since other endpoints accept them with different format.

Reply:
It won't accept multiple values. We've found a bug in this endpoint (https://piwikpro.atlassian.net/browse/TM-9231). It will be fixed as soon as possible :)

Reply:
Then you get green light from me.